### PR TITLE
cython

### PR DESCRIPTION
**Pull request by msarahan**
Hopefully this one is quick and easy.

### CODE REVIEW

File: recipes/cython/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  description: |
+    Cython is an optimising static compiler for both the Python programming
+    language and the extended Cython programming language. It makes writing C
+    extensions for Python as easy as Python itself.
+  doc_url: http://cython.org/#documentation
+  dev_url: https://github.com/cython/cython
+
+extra:
+  recipe-maintainers:
+    - msarahan
```
Comment (jakirkham) [inline recipes/cython/meta.yaml]: Please add me. 😄  Also, nit: make sure there is a terminal newline.

File: recipes/cython/osx-post-link.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $PREFIX/lib
+rm -f libgcc_s.10.5.dylib
+ln -s /usr/lib/libgcc_s.1.dylib libgcc_s.10.5.dylib
```
Comment (jakirkham) [inline recipes/cython/osx-post-link.sh]: What's up with this post-link script?

File: recipes/cython/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  commands:
+    - cython --version
+  files:
+    - fib.pyx
+
+about:
+  home: http://www.cython.org/
+  license: Apache
```
Comment (jakirkham) [inline recipes/cython/meta.yaml]: Version?

Comment (ocefpaf) [inline recipes/cython/meta.yaml]: It is `Apache-2.0`.

File: recipes/cython/osx-post-link.sh (hunk 2 above)
Comment (msarahan) [inline recipes/cython/osx-post-link.sh]: yeah, it reeks pretty bad.  I'm going to leave it out and keep fingers crossed.

Comment (jakirkham) [inline recipes/cython/osx-post-link.sh]: Well at least you have a decent test case to see if this is an issue. With any luck, this is an ancient workaround for an issue already fixed by upstream.

File: recipes/cython/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  entry_points:
+    - cython = Cython.Compiler.Main:setuptools_main
+    - cythonize = Cython.Build.Cythonize:main
+    - cygdb = Cython.Debugger.Cygdb:main
+
+requirements:
+  build:
+    - python
+    - setuptools       # [win]
```
Comment (jakirkham) [inline recipes/cython/meta.yaml]: Why Windows only?

File: recipes/cython/build.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+$PYTHON setup.py install
+
+if [ `uname` == Darwin ]
+then
+    cp $RECIPE_DIR/osx-post-link.sh $PREFIX/bin/.cython-post-link.sh
+fi
```
Comment (jakirkham) [inline recipes/cython/build.sh]: Might need to cut this too.

**Comment by jakirkham** [comment]
LGTM. Thanks @msarahan.

Also, nice to see that Mac patch was dropped without incident. Will wait for AppVeyor and then merge.

**Comment by jakirkham** [comment]
Just to confirm, there is nothing else you are wanting to do to the recipe before it is merged, @msarahan? Asking because of the WIP in the title.

**Comment by msarahan** [comment]
Nope.  Good by me.  AppVeyor seems like it might have reported success during rate limiting, though.  It's all green over there, but still yellow here.

**Comment by jakirkham** [comment]
Weird. Would be good to report this issue to them.

**Comment by jakirkham** [comment]
Thanks @msarahan.

**Comment by jakirkham** [comment]
@scoder @robertwb @dalcinl @larsmans @insertinterestingnamehere @jdemeyer , just letting you know that we have added `cython` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge cython`]( http://anaconda.org/conda-forge/cython ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/cython-feedstock ). If you or others have any interest in being added as a maintainer (useful for making new releases for instance), we would be more than happy to add you to the maintainers list. Thanks for your invaluable contributions to `cython`.

**Pull request merged**